prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml

```diff
@@ -119,3 +119,52 @@ presubmits:
       - name: codecov
         secret:
           secretName: codecov
+
+  GoogleCloudPlatform/guest-logging-go:
```

Comment:
this should live under GoogleCloudPlatform/guest-logging-go subdir

Reply:
i'm fine with splitting out but confused why this is the change that prompts it, it's the third repo added to this file. if the dirs under GoogleCloudPlatform should match repo names, then this dir should be renamed from gcp-guest to osconfig, and guest-agent jobs moved out.

if that's what you have in mind, can we do GoogleCloudPlatform/gcp-guest/<repo> dirs, so we can put OWNERS at the gcp-guest level?

Reply:
Technically there's no hard rules, we collect all *.yaml files under the prowjobs/ subdir. I just feel like split out jobs by repos will help you keep your config files reviewable and making more granular changes. Feel free to address it later.